bin/tictactoe

```diff
@@ -1,3 +1,55 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+### Different game initializations available ###
+
+def start
```

Comment:
I would convert this method into a CLI class with smaller methods that lives in the `lib` folder. You will then be able to just call this on line 5 

```ruby 
CLI.new.start 
```